packages/oscal-react-library/src/components/OSCALCatalogBaseline.tsx

```diff
@@ -0,0 +1,1518 @@
+export const CatalogBreadCrumbsMenu: React.FC<OSCALModel | undefined> = (item) => {
+  if (item !== undefined && item.model.title !== undefined && item.model.title.length > 0)
+    return (
+      <OSCALBreadCrumbs
+        aria-label="breadcrumb"
+        sx={{
+          ":hover": {
+            color: "#FF6600",
+          },
+        }}
+      >
+        <OSCALLink
+          href="/"
+          sx={{
+            ":hover": {
+              color: "#FF6600",
+            },
+          }}
+        >
+          HOME
+        </OSCALLink>
+        <OSCALLink
+          href="/"
+          sx={{
+            fontWeight: "700",
+            ":hover": {
+              color: "#FF6600",
+            },
+          }}
+        >
+          CATALOGS & BASELINES
+        </OSCALLink>
+        <OSCALLink
+          href="/"
+          sx={{
+            color: "#002867",
+            fontWeight: "700",
+            ":hover": {
+              color: "#FF6600",
+            },
+          }}
+        >
+          {" "}
+          {item.model.title.toUpperCase()}
+        </OSCALLink>
+      </OSCALBreadCrumbs>
+    );
+  else
+    return (
+      <OSCALBreadCrumbs
+        aria-label="breadcrumb"
+        sx={{
+          ":hover": {
+            color: "#FF6600",
+          },
+        }}
+      >
+        <OSCALLink
+          href="/"
+          sx={{
+            ":hover": {
+              color: "#FF6600",
+            },
+          }}
+        >
+          HOME
+        </OSCALLink>
+        <OSCALLink
+          href="/"
+          sx={{
+            color: "#002867",
+            fontWeight: "700",
+            ":hover": {
+              color: "#FF6600",
+            },
+          }}
+        >
+          CATALOGS & BASELINES
+        </OSCALLink>
+      </OSCALBreadCrumbs>
+    );
+};
+
+export function CatalogBaselineTabs() {
+  const [value, setValue] = React.useState("1");
+
+  const handleChange = (event: React.SyntheticEvent, newValue: string) => {
+    setValue(newValue);
+  };
+
+  return (
+    <Box
+      sx={{
+        width: 1070,
+        height: 900,
+        typography: "body1",
+        top: 200,
+        left: 100,
+        position: "absolute",
+        background: "linear-gradient(0deg, #FFFFFF, #FFFFFF)",
+      }}
+    >
+      <TabContext value={value}>
+        <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
+          <TabList
+            onChange={handleChange}
+            aria-label="lab API tabs example"
+            sx={{ background: "#F6F6F6" }}
+          >
+            <CTab
+              label="Controls"
+              value="1"
+              sx={{
+                ":focus": {
+                  background: "#FFffff",
+                  weight: 600,
+                },
+              }}
+            />
+            <CTab
+              label="Catalog Details"
+              value="2"
+              sx={{
+                ":focus": {
+                  background: "#FFffff",
+                  weight: 600,
```

Comment:
This sx `focus` styling also looks common to `CTabs`. Can this also be placed in the styled component above?